packages/sources/finage/src/config.ts

```diff
@@ -1,16 +1,34 @@
 import { Requester, util } from '@chainlink/ea-bootstrap'
-import { Config } from '@chainlink/types'
+import { Config as BaseConfig } from '@chainlink/types'
 
 export const NAME = 'FINAGE'
 
 export const DEFAULT_BASE_URL = 'https://api.finage.co.uk'
-export const DEFAULT_WS_API_ENDPOINT = 'wss://e4s39ar3mr.finage.ws:7002'
+export const DEFAULT_STOCK_WS_API_ENDPOINT = 'wss://e4s39ar3mr.finage.ws:7002'
+export const DEFAULT_FOREX_WS_API_ENDPOINT = 'wss://w29hxx2ndd.finage.ws:8001'
+export const DEFAULT_ENDPOINT = 'stock'
+
+export const ENV_STOCK_WS_API_ENDPOINT = 'STOCK_WS_API_ENDPOINT'
+export const ENV_FOREX_WS_API_ENDPOINT = 'FOREX_WS_API_ENDPOINT'
+
+export type Config = BaseConfig & {
+  stockWsEndpoint: string
+  forexWsEndpoint: string
+}
 
 export const makeConfig = (prefix?: string): Config => {
   const config = Requester.getDefaultConfig(prefix, true)
-  config.api.baseURL = config.api.baseURL || DEFAULT_BASE_URL
-  const wsUrl = config.api.baseWsURL || DEFAULT_WS_API_ENDPOINT
+  const stockWsUrl = util.getEnv(ENV_STOCK_WS_API_ENDPOINT) || DEFAULT_STOCK_WS_API_ENDPOINT
```

Comment:
We don't have a precedence for this yet, but we should document these environment variables in the README. Maybe a new WS section.

(also add to the schema)